app/components/main-content/PrelaunchSignUpForm.tsx

```diff
@@ -31,21 +24,9 @@ const PrelaunchSignUpForm: React.FC<PrelaunchSignUpFormProps> = ({ setFlowState
     validate: validationRules
   });
 
-  useEffect(() => {
-    async function checkRedirectResult() {
-      const res = await getRedirectResult(auth);  // Needed to access user data after redirect during OAuth sign in
-
-      if (res) {
-        setFlowState('processing');
-        await createUserDoc(res.user);
-        setFlowState('confirmed')
-      }
-    }
-    checkRedirectResult();
-  }, [setFlowState]);
 
   async function handleSubmit(values: FormValues) {
-    const { email, name } = values;
+    const { email, name, interests, source, features } = values;
```

Comment:
Updates related to Firestore are implemented in #39, unless it is causing errors it may be better to leave it to the other ticket and not make changes in this PR.

Reply:
Thanks!